### PR TITLE
Design fixes, bump react-coin-icon

### PR DESCRIPTION
**Pull request by christianbaroni**
ui fixes + some design updates

before → after:
Screen Shot 2020-12-16 at 12 36 56 PM

### CODE REVIEW

File: src/components/buttons/PasteAddressButton.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default function PasteAddressButton({ onPress }) {
       testID="paste-address-button"
       {...(android && { height: 30, overflowMargin: 15, width: 60 })}
     >
-      <Text color="white" weight="semibold">
+      <Text color="white" weight="bold">
```
Comment (osdnk) [inline src/components/buttons/PasteAddressButton.js:50]:
```suggestion
      <Text color="white" weight={fonts.weight.semibold}>
```

File: src/components/buttons/MiniButton.js [hunk 2]
```diff
@@ -56,9 +50,9 @@ export default function MiniButton({
           shadows={disabled ? shadows.disabled : shadows.default}
           width={width}
         />
-        <Content>
+        <Content hasLeadingIcon={hasLeadingIcon}>
           {typeof children === 'string' ? (
-            <Text color="white" weight="semibold">
+            <Text align="center" color="white" weight="bold">
```
Comment (osdnk) [inline src/components/buttons/MiniButton.js:55]:
```suggestion
            <Text align="center" color="white" weight={fonts.weight.semibold}>
```

File: src/components/savings/SavingsSheetEmptyState.js [hunk 3]
```diff
@@ -82,6 +82,7 @@ const SavingsSheetEmptyState = ({
           label="􀁍 Deposit from Wallet"
           onPress={onDeposit}
           size="big"
+          weight="bold"
```
Comment (osdnk) [inline src/components/savings/SavingsSheetEmptyState.js:85]:
```suggestion
          weight={fonts.weight.semibold}
```

File: src/screens/RestoreSheet.js [hunk 4]
```diff
@@ -72,13 +72,17 @@ export default function RestoreSheet() {
...
+    InteractionManager.runAfterInteractions(() => goBack());
+    InteractionManager.runAfterInteractions(() => {
+      setTimeout(() => navigate(Routes.IMPORT_SEED_PHRASE_FLOW), 50);
+    });
   }, [goBack, navigate]);
 
   const onWatchAddress = useCallback(() => {
-    goBack();
-    navigate(Routes.IMPORT_SEED_PHRASE_FLOW);
+    InteractionManager.runAfterInteractions(() => goBack());
```
Comment (osdnk) [inline src/screens/RestoreSheet.js:82]:
```suggestion
    InteractionManager.runAfterInteractions(goBack};
```

File: src/screens/RestoreSheet.js [hunk 5]
```diff
@@ -72,13 +72,17 @@ export default function RestoreSheet() {
   }, [setParams]);
 
   const onManualRestore = useCallback(() => {
-    goBack();
-    navigate(Routes.IMPORT_SEED_PHRASE_FLOW);
+    InteractionManager.runAfterInteractions(() => goBack());
```
Comment (osdnk) [inline src/screens/RestoreSheet.js:75]:
```suggestion
    InteractionManager.runAfterInteractions(goBack);
```

**Review by osdnk — APPROVED** [review]

File: src/components/buttons/MiniButton.js (hunk 2 above)
Comment (christianbaroni) [reply src/components/buttons/MiniButton.js:55]: @osdnk is this necessary?

think our text component handles font weights

cc @mikedemarais

Comment (mikedemarais) [reply src/components/buttons/MiniButton.js:55]: hey! yes our `<Text />` component accepts all the keys of our `fonts.weights` as props

i know that we are pretty inconsistent about this using this API, and other style/layout APIs i’ve made — i definitely think the APIs can be improved and that we can consolidate patterns

but i agree with @christianbaroni that it’s simpler to just use “bold” here 

very down to discuss / flesh out better style APIs

cc @osdnk 

also i notice that michał’s suggestion not only changes the syntax but also from “bold” to “semibold” — not sure if the weight change was intentional or not (cc @christianbaroni) or if he was just intending to suggest syntax change 

😘

**Comment by mikedemarais** [comment]
fuuuuuck i clicked the wrong button and accidentally committed the wrong suggestion @christianbaroni @osdnk 

i’m on phone right now afk can someone please revert

File: src/screens/RestoreSheet.js (hunk 5 above)
Comment (mikedemarais) [reply src/screens/RestoreSheet.js:75]: yes

File: src/screens/RestoreSheet.js (hunk 4 above)
Comment (mikedemarais) [reply src/screens/RestoreSheet.js:82]: yes

**Review by skylarbarrera — APPROVED** [review]
looks great 👌

**Pull request merged**